ui/pages/swaps/view-quote/view-quote.js

```diff
@@ -840,7 +859,33 @@ export default function ViewQuote() {
                 onSubmit={(aggId) => dispatch(swapsQuoteSelected(aggId))}
                 swapToSymbol={destinationTokenSymbol}
                 initialAggId={usedQuote.aggregator}
-                onQuoteDetailsIsOpened={quoteDetailsOpened}
+                onQuoteDetailsIsOpened={trackEvent({
```

Comment:
[REQ] I missed this on the first go around, this needs to be a function
```js
() => trackEvent({
```
otherwise its invoked on every render

Reply:
@brad-decker Done [here](https://github.com/MetaMask/metamask-extension/pull/13955/commits/dd1dc78a3a270553def9265328d70cc32e8c3ae3)